inspirehep/records/api/mixins.py

```diff
@@ -29,3 +31,18 @@ def citation_count(self):
             record.
         """
         return self._citation_query().count()
+
+    @property
+    def citations_by_year(self):
+        """Builds dict with cittation summary by year for current record
```

Comment:
* typo: `citation`. 
* `Builds dict` is not nice as it doesn't build anything persistent, maybe
```suggestion
        """Return the number of citations received per year for the current record.
```